pkg/rules/slp085.go

```
@@ -0,0 +1,104 @@
+
+func (SLP085) ID() string                { return "SLP085" }
+func (SLP085) DefaultSeverity() Severity { return SeverityBlock }
+func (SLP085) Description() string {
+	return "SQL query built via string concatenation - use parameterized queries with placeholders"
+}
+
+var (
+	// Common SQL DML keywords
+	slp085SqlKeywords = []string{"SELECT", "INSERT", "UPDATE", "DELETE", "DROP", "CREATE", "ALTER", "TRUNCATE"}
+	// SQL-like patterns that could indicate injection risk
+	slp085SqlConcatPattern = regexp.MustCompile(`(?i)\b(?:SELECT|INSERT|UPDATE|DELETE|DROP|CREATE|ALTER|TRUNCATE)\b.*[\+\+]|(\w+_(?:query|execute|exec|run|statement))\s*\([^)]*[\+\+][^)]*\)`)
```

golangci-lint (2.11.4)

[error] var slp085SqlConcatPattern is unused

(unused)